packages/frontend/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -666,7 +666,7 @@
 	"dataMapping.tableView.tableColumnsExceeded": "Some columns are hidden",
 	"dataMapping.tableView.tableColumnsExceeded.tooltip": "Your data has more than {columnLimit} columns so some are hidden. Switch to {link} to see all data.",
 	"dataMapping.tableView.tableColumnsExceeded.tooltip.link": "JSON view",
-	"dataMapping.schemaView.emptyData": "No fields - item(s) exist, but they're empty",
+	"dataMapping.schemaView.emptyData": "[Execute previous nodes] to see schema",
```

Comment:
The execute previous nodes part should be clickable, you can use the `i18n-t` component

Example:
```html
<i18n-t tag="span" keypath="ndv.input.rootNodeHasNotRun.description">
	<template #link>
		<a
			href="#"
			data-test-id="switch-to-mapping-mode-link"
			@click.prevent="onInputModeChange('mapping')"
			>{{ i18n.baseText('ndv.input.rootNodeHasNotRun.description.link') }}</a
		>
	</template>
</i18n-t>
```

Might also be worth it to use the `NodeExecuteButton` component, it makes sure the button/link is disabled when needed etc. Example on how to style it like a link:

```
<NodeExecuteButton
			text
			hide-icon
			transparent
			...
		/>
```